routes/callback.ts

```diff
@@ -0,0 +1,32 @@
+import express from 'express'
+import { sgidService } from '../services/sgid-client.service'
+import { formatData } from '../utils'
+import { RANDOMNONCE } from '../config'
+import { nodeCache } from '../services/node-cache.service'
+import { SESSION_COOKIE_NAME } from '../constants'
+
+/**
+ * Main controller function to generate the callback page
+ */
+export const callback = async (req: express.Request, res: express.Response) => {
+  try {
+    const { code, state } = req.query
+    const codeVerifier = nodeCache.get(req.cookies[SESSION_COOKIE_NAME])
```

Comment:
If our concern here is not abiding to best practices, maybe we can do the same thing in our developer docs, ie. an explicit comment:

```
/**
 * In-memory store for session data.
 * In a real application, this would be a database.
 */
```

Reply:
Keeping this demo app as a single instance. Added comments as suggested by kok seng. Resolved in [73aca9f](https://github.com/opengovsg/sgid-test-app/pull/94/commits/73aca9fd057c093722ee6f95ba31f347607f4197).